nixos/modules/services/networking/thelounge.nix

```
@@ -72,4 +74,6 @@ in {
 
     environment.systemPackages = [ pkgs.thelounge ];
   };
+
+  imports = [ (mkRemovedOptionModule [ "private" ] [ "public" ]) ];
```

```suggestion
  imports = [ (mkRemovedOptionModule [ "services" "thelounge" "private" ] "The option was renamed to `services.thelounge.public`, to follow upstream.") ];
```

The usage here is wrong. This looks for an option with a fully qualified name `private`. Additionally, it tries to coerce a list `[ "public" ]` into the error message.